src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -82,6 +91,7 @@ public void execute() {
                      .ifPresent(path -> export(path, fileDialogConfiguration.getSelectedExtensionFilter(), exporters));
     }
 
+    @SuppressWarnings("checkstyle:EmptyLineSeparator")
```

Comment:
I know that checkstyle can be annoying, but it helps to maintain a single code style across the project and prevents merge conflicts due to whitespaces etc
So please do not remove the warnings ;)